supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-virtualization.adoc

```diff
@@ -101,15 +96,14 @@ Tabs in the details pane are always referred to as, for example, "the General ta
 [discrete]
 [[host-rhv]]
 ==== image:images/yes.png[yes] host (noun)
-*Description*: Hosts are servers that provide the processing capabilities and memory resources used to run virtual machines. There are two types of hosts: Red Hat Enterprise Linux hosts, and Red Hat Enterprise Virtualization Hypervisor (in version 3.x) or Red Hat Virtualization Host (in version 4.x).
-
-Use "host" to refer to hosts in general, not "hypervisor", "hypervisor host", or "virtualization host". When referring to a specific type of host, use "Red Hat Enterprise Linux host", "Red Hat Enterprise Virtualization Hypervisor host", or "Red Hat Virtualization Host".
+*Description*: Hosts are servers that provide the processing capabilities and memory resources used to run virtual machines. There are two types of hosts: Red Hat Enterprise Linux hosts and Red Hat Virtualization Host.
```

Comment:
If `Red Hat Virtualization Host` is singular here, does it need an indefinite article 'a' before it?

Reply:
Or maybe it should be 'Hosts' plural? Or maybe it's fine :) just getting tripped up on 'two types' then the singular-plural disagreement reads weirdly.